### PR TITLE
gitleaks.io

### LINKED ISSUE #4180: +gitleaks
https://github.com/gitleaks/gitleaks
Homebrew Formula: https://formulae.brew.sh/formula/gitleaks

### PR DESCRIPTION
**Pull request by and-ri**
closes #4180

### CODE REVIEW

File: projects/gitleaks.io/package.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - go build $GO_ARGS -ldflags="$LDFLAGS"
+  env:
+    LDFLAGS: -X github.com/zricethezav/gitleaks/v{{version.major}}/cmd.Version={{version}}
+    GO_ARGS:
+      - -trimpath
+      - -o="{{prefix}}/bin/gitleaks"
+    linux:
+      LDFLAGS:
+        - -buildmode=pie
+test: gitleaks version | grep {{version}}
```
Comment (jhheider) [inline projects/gitleaks.io/package.yml]: will gitleaks let us do a better test?

Comment (and-ri) [reply projects/gitleaks.io/package.yml]: > will gitleaks let us do a better test?

Yes I will do a test with git

**Comment by tsirysndr** [comment]
missing `provides`:
```yaml
provides:
   - bin/gitleaks
```

**Comment by and-ri** [comment]
> missing `provides`:
> 
> ```yaml
> provides:
>    - bin/gitleaks
> ```

fixed by #4189 👌🏻

**Pull request merged**